internal/server/config.go

```diff
@@ -612,10 +616,38 @@ func (Server) loadProvider(db *gorm.DB, input Provider) (*models.Provider, error
 			URL:          input.URL,
 			ClientID:     input.ClientID,
 			ClientSecret: models.EncryptedAtRest(input.ClientSecret),
+			AuthURL:      input.AuthURL,
+			Scopes:       input.Scopes,
 			Kind:         kind,
 			CreatedBy:    models.CreatedBySystem,
 		}
 
+		if provider.Kind != models.InfraKind {
+			// only call the provider to resolve info if it is not known
+			if input.AuthURL == "" && len(input.Scopes) == 0 {
```

Comment:
In config the auth URL and scopes can be specified to skip the external look-up.